tests/tableediting.js

```diff
@@ -294,6 +294,22 @@ describe( 'TableEditing', () => {
 				] ) );
 			} );
 
+			it( 'should not create another row and not move the caret if insertTableRowBelow command is disabled', () => {
+				setModelData( model, modelTable( [
+					[ '11', '[12]' ]
```

Comment:
maybe test should include also testing the selection so it should start with collapsed and should stay collapsed (not changed) after the <kbd>tab</kbd>.